setup.py

```diff
@@ -11,7 +11,7 @@
                 ]
 
     cellpose_deps = [
-        "cellpose>=1.0.2"
```

Comment:
@sugan89 we need both pins still, please and thank you!

Reply:
Thanks! I have made the change. Please let me know if this is okay.